2018/ffs-wp.md

```diff
@@ -0,0 +1,277 @@
+- Usability
+- Features
+- Stabilität
+- Dokumentation
+
+ Mit freien dezentralen Diensten (z.B. Jabber als Ersatz für WhatsApp) verhält es sich ähnlich.
+ Der Einfachheit halber geht der folgende Text erstmal nur auf "klassische Software" ein,
+ inbesondere Desktop-Applikationen.
+
+
+**Warum ist proprietäre Software in den genannten Punkten oft besser?**
+Geschäftsmodell (Verkauf von Software uder Daten) generiert Einnahmen.
+Dafür kann Arbeitskraft und Aufmerksamkeit eingekauft werden.
+
+**Warum funktioniert das bei Freier Software so nicht?**
+Diese kann auf Grund der wesensbestimmenden Freiheit nicht "verkauft" werden.
+
+**Warum funktioniert freie Software dann überhaupt?**
+Weil für manche Menschen/Organisationen nicht nur kurzfristige Nutzenmaximierung handlungsleitend ist.
+Konkrete Quellen von Ressourcen:
+
+- Arbeit an Freier Software als Hobby (Sinnstiftung durch Problemlösen, Zusammenarbeit mit anderen, Anerkennung durch positives Feedback, Lerneffekt, Demonstration von Fähigkeiten (z.B. für Bewerbungsverfahren))
+- Durch Unternehmen oder Behörden bezahlte Entwicklung (motiviert durch Eigeninteresse (Einsicht in die Vorteile))
+- Spenden (Altruismus oder aufgeklärtes Eigeninteresse)
+- Tragfähige [Geschäftsmodelle](https://en.wikipedia.org/wiki/Business_models_for_open-source_software)
+
+Fakt ist aber: in den allermeisten Projekten fehlt es an Ressourcen für eine wünschenswerte Weiterentwicklung.
+Mit anderen Worten: obige Quellen von Ressourcen sind nicht ausreichend.
+
+
+## Analyse existierender Geschäftsmodelle
+
+Wikipedia listet folgende [Geschäftsmodelle](https://en.wikipedia.org/wiki/Business_models_for_open-source_software)
+für "Open-Source-Software" auf:
+
+    3.1 Dual-licensing
+    3.2 Selling professional services
+    3.3 Selling of branded merchandise
+    3.4 Selling of certificates and trademark use
+    3.5 Selling software as a service
+    3.6 Partnership with funding organizations
+    3.7 Voluntary donations
+    3.8 Bounty driven development
+    3.9 Pre-order/crowdfunding/reverse-bounty model
+    3.10 Crowdsourcing
+    3.11 Advertising-supported software
+    3.12 Selling of optional proprietary extensions
+    3.13 Selling of required proprietary parts of a software product
+    3.14 Selling of proprietary update systems
+    3.15 Re-licensing under a proprietary license
+    3.16 Obfuscation of source code
+    3.17 Delayed open-sourcing
+    3.18 Open sourcing on end-of-life
+
+
+ Für die Projekte mit der Zielgruppe der Endbenutzer\*innen kommen bestimmte Modelle
+ nicht in Frage bzw. sind nicht wünschenswert (Unterschied zwischen Open-Source und freier Software).
+ Im Folgenden sind Ausschlussgründe und die davon betroffenen Geschäftsmodelle aufgelistet.
+
+
+- Nicht sinnvoll, Weil kein Weiterverkauf angestrebt wird
+    - Dual-licensing
+    - Re-licensing under a proprietary license
+- Zielkonflikt: Perfekte Software braucht keinen Support/Schulung
+    - Selling professional services
```

Comment:
Ich glaube das ist auf dem Desktop vor allem deswegen irrelevant, weil kein normaler Endbenutzer für den Support zahlen wird. Eher schon für Updates oder ein Accountabo oder Storage Space oder sonstwas, je nach Dienst halt.

Tatsächlich kann das gerade bei Kommunikationsdiensten garnicht mal schlecht funktionieren. Ich sehe das Problem (zumindest für mich persönlich) auch durchaus in dem bürokratischen Aufwand / Unsicherheiten bei Steuerdingen.

Reply:
Wenn ich betrachte wieviele Leute Literatur für Eord/Excel oder Windows-Tipps daheim stehen haben, würde ich das komplett bestreiten. Auch „Nachbarschaftshilfe“ (bzw. teils gegen kleines Entgelt) ist völlig normal. Und es geht ja nicht um Privatanwender, sondern um Firmen mit ordentlichen Anforderungen die auch gerne ihr Risiko durch entsprechende Dienstleistungen mindern. Wichtig: Die Dienstleistungen sollten über den Betriebszeitraum kostengünstiger sein als das ermittelte Risiko.

Reply:
Ein Problem bei dem Geschäftsmodell "Services" ist: es bindet massiv Arbeitskraft. Statt einer Person zu erklären wie sie ein bestimmtes LaTeX-Problem beheben kann, wäre es effizienter daran zu arbeiten die Fehlermeldungen so zu gestalten, dass sich die Frage von selber löst. Also Support-Dienstleistungen haben schon ihren Sinn, aber das KnowHow von Projektinsidern ist dafür IMHO zu wertvoll.

Reply:
FWIW, als ich schrieb "Tatsächlich kann das gerade bei Kommunikationsdiensten garnicht mal schlecht funktionieren" dachte ich insbesondere an das was Conversations tut (Infrastruktur mit vermieten).